### DISCUSSION TITLE
Do not translate words but for specific websites

### DISCUSSION THREAD

**Discussion by ghost** [immersive-translate/immersive-translate#1958] [Ideas]
### 新功能 | New Feature

1. Add an option to exclude or include these websites under or above the translate these words section.

### 为什么您想要添加这些功能 | Why you want to add these features

1. Why this feature is necessary: I think this feature is very important, if you ask why, when reading the wiki of a specific source, x term can be translated very differently than it is normally translated and when you read it you may not understand it, you may not grasp it., let me give you an example: As a Chinese person, let's say you play a global game with a lot of players, for example league of legends, words like zoning, push, shove, totem, farming, etc. are quite different from their normal meaning in this game (in terms of translation), so it would be more logical not to translate them on a site about league of legends, which is easier to understand because you hear those terms all the time than reading the translation because it usually doesn't translate them in that sense since translator apps translates directly with the first meaning that comes to mind, but when you read something else from another site other than league of legends or any book etc., it will give you the literal translation of the terms I mentioned above, so you should translate those words there to the unity of meaning is preserved, sorry if it is not understanble, too sleep-deprived.

### 提议 | Proposal

In the "Don't translate these words" section, I think an option could be added such as only on these sites or outside these sites

**Comment by keyboard3** [comment]
I understand the need, but the need may be small, but it is more complex. We need to discuss it internally.

**Reply by ghost** [reply]
do we have any new about it?

**Reply by theowenyoung** [reply]
Hello, we currently have no plans to support this in our roadmap

**Reply by ghost** [reply]
ty
